src/osx/cocoa/stdpaths.mm

```diff
@@ -139,4 +139,13 @@ ConfigFileConv WXUNUSED(conv)) const
     return fn.GetFullName();
 }
 
+wxString wxStandardPaths::GetSharedLibrariesDir() const
+{
+    // Shared libraries on OSX should be stored inside the
+    // <Bundle.app>/Contents/Frameworks
+    wxFileName fn( GetExecutablePath() );
+    fn.RemoveLastDir();
+    return fn.GetPathWithSep() + "Frameworks/";
```

Comment:
I think we need one of the functions from [this page](https://developer.apple.com/documentation/foundation/nsbundle/1408900-builtinpluginspath/) instead although I'm not sure which one. Please try using them to see which of them returns `Frameworks`.

Reply:
@vadz ,
Found it.
Tested and committed.

Thx